contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1847,13 +1854,15 @@ export const Task = new IndexedDBResource({
       task.channel_id = task.channel_id.replace('-', '');
     }
     return this.transaction({ mode: 'rw', source: IGNORED_SOURCE }, () => {
-      return this.table
-        .where(this.idField)
-        .noneOf(tasks.map(t => t[this.idField]))
-        .delete()
-        .then(() => {
-          return this.table.bulkPut(tasks);
-        });
+      let deletePromise = Promise.resolve();
```

Comment:
I think this logic is preserving a single task of the type - rather than a single task per type per channel? This might cause weird behaviour if you're editing two channels at once.